composer.json

```diff
@@ -20,6 +20,7 @@
 	"require":{
 		"php":">=7.2",
 		"contao/core-bundle":"^4.9",
+		"doctrine/dbal": "^3.3",
```

Comment:
I think this won't work with 4.9 - did you try this?

Reply:
Hmm, indeed it won't: https://github.com/contao/contao/blob/4.9/core-bundle/composer.json#L49

Should we bump the extension to 4.13 then? Otherwise we'd have to use some `method_exists()` magic to check the DBAL version.